src/development/LocalTest/Services/LocalApp/Implementation/LocalAppHttp.cs

```diff
@@ -79,5 +88,31 @@ public LocalAppHttp(IHttpClientFactory httpClientFactory, IOptions<LocalPlatform
 
             return ret;
         }
+
+        public async Task<Instance?> Instantiate(string appId, Instance instance, string xmlPrefill, string xmlDataId)
+        {
+            var requestUri = $"{appId}/instances";
+            var serializedInstance = JsonSerializer.Serialize(instance, new JsonSerializerOptions { DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingDefault });
+
+            var content = new MultipartFormDataContent();
+            content.Add(new StringContent(serializedInstance, System.Text.Encoding.UTF8, "application/json"), "instance");
+            if (!string.IsNullOrWhiteSpace(xmlPrefill) && xmlDataId is not null)
+            {
+                content.Add(new StringContent(xmlPrefill, System.Text.Encoding.UTF8, "application/xml"), xmlDataId);
+            }
+
+            using var message = new HttpRequestMessage(HttpMethod.Post, requestUri);
+            message.Content = content;
+            // TODO: Figure out how to get orgnumber for app owner from appId
```

Comment:
Remove the comment now that a method for obtaining the organization number has been found?

Reply:
```suggestion
```